server/db/models/swag.js

```diff
@@ -1,12 +1,23 @@
 'use strict';
 var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
+var Promise = require('bluebird');
 
 var schema = new Schema({
 	name: String,
 	category: String,
 	price: Number,
-	imageUrl: String
+	imageUrl: String, 
+  height: String,
+  width: String,
+  left: String,
+  top: String
 });
```

Comment:
This is so weird, the indentation looks fine on my version, but you should probably change it.. hahaha